static/tsconfig.json

```diff
@@ -5,7 +5,7 @@
         "name": "typescript-tslint-plugin"
       }
     ],
-    "lib": ["dom", "ES2021"],
+    "lib": ["dom", "es2021"],
```

Comment:
nice! can you add a comment for the nuance?

Reply:
Looked into this a bit more, and looks like it was actually a local issue with my VS Code settings (needed to update). Removed this change.